zkevm-circuits/src/table.rs

```diff
@@ -1099,7 +1099,10 @@ impl CopyTable {
     pub fn construct<F: Field>(meta: &mut ConstraintSystem<F>, q_enable: Column<Fixed>) -> Self {
         Self {
             is_first: meta.advice_column(),
-            id: meta.advice_column_in(SecondPhase),
+            id: word::Word::new([
+                meta.advice_column_in(FirstPhase),
```

Comment:
nitpick: `meta.advice_column()` is sufficient

Reply:
Nice catch! I didn't realize that.